system/index.js

```diff
@@ -131,8 +131,9 @@ System.prototype.isUserRegistrationEnabled = function() {
  * @return {boolean}
  */
 System.prototype.hasLegalTerms = function() {
-    try {
-        return this.getConfig(['legal', 'terms']) !== false;
+	let config = this.getConfig(['legal', 'terms']);
```

Comment:
we're now mixing var and let/const statements :) We should probably stick to one? :)